Wrong argument type within ForceTemplateParsingInTsfeAndTemplateServiceRector

Got the following error:

```
[ERROR] Could not process "[redacted]" file by
         "Rector\Core\Rector\AbstractRector", due to:
         "Argument 1 passed to Rector\Core\Rector\AbstractRector::isName() must implement interface PhpParser\Node,
         string given, called in
         vendor/ssch/typo3-rector/src/Rector/v10/v0/ForceTemplateParsingInTsfeAndTemplateServiceRector.php:67".
```

--- a/src/Rector/v10/v0/ForceTemplateParsingInTsfeAndTemplateServiceRector.php
+++ b/src/Rector/v10/v0/ForceTemplateParsingInTsfeAndTemplateServiceRector.php
@@ -64,7 +64,7 @@ final class ForceTemplateParsingInTsfeAndTemplateServiceRector extends AbstractR
                         Typo3NodeResolver::TYPO_SCRIPT_FRONTEND_CONTROLLER)
                 )
             ) &&
-            (property_exists($node, 'name') && $this->isName($node->name, 'forceTemplateParsing'));
+            (property_exists($node, 'name') && $this->isName($node, 'forceTemplateParsing'));
     }
 
     /**

